From b42cc9467d522246bec3d9abc6e820c9e8f68743 Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Fri, 30 Aug 2013 08:51:21 +0200
Subject: [PATCH 02/29] Remove i82550 network card emulation

RH-Author: Miroslav Rezanina <mrezanin@redhat.com>
Message-id: <8a443ebad7bda82e3a609db476db3f77b97dab53.1377851454.git.mrezanin@redhat.com>
Patchwork-id: 53933
O-Subject: [RHEL7 qemu-kvm PATCHv2 1/6] Remove i82550 network card emulation
Bugzilla: 921983
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Michal Novotny <minovotn@redhat.com>

From: Miroslav Rezanina <mrezanin@redhat.com>

Bugzilla: 921983
Upstream: n/a, RHEL only

We do not support i82550 card and it's variants. Removing the card
from configuration.

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 default-configs/pci.mak | 1 -
 1 file changed, 1 deletion(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 default-configs/pci.mak |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/default-configs/pci.mak b/default-configs/pci.mak
index c0fb0ae..938f413 100644
--- a/default-configs/pci.mak
+++ b/default-configs/pci.mak
@@ -4,7 +4,6 @@ CONFIG_VIRTIO=y
 CONFIG_USB_UHCI=y
 CONFIG_USB_EHCI=y
 CONFIG_USB_XHCI=y
-CONFIG_EEPRO100_PCI=y
 CONFIG_AC97=y
 CONFIG_HDA=y
 CONFIG_RTL8139_PCI=y
-- 
1.7.1

